documentation/src/docs/key-features.md

```diff
@@ -10,86 +10,85 @@ You may obtain a copy of the License at
 # Pravega Key Features
 
 This document explains some of the key features of Pravega.  It may be
-advantageous if you are already familiar with the core [concepts of
-Pravega](pravega-concepts.md).
+advantageous if you are already familiar with the core [Pravega Concepts](pravega-concepts.md).
 
 ## Pravega Design Principles
 
-Pravega was designed to support the new generation of streaming applications:
-applications that deal with a large amount of data arriving continuously that
-also need to generate an accurate analysis of that data in the face of late
-arriving data, data arriving out of order and failure conditions. There are
-several open source tools to enable developers to build such applications,
-including Apache Flink, Apache Beam, Spark Streaming and others.
-To date, these applications used systems such as Apache Kafka, Apache ActiveMQ,
-RabbitMQ, Apache Cassandra, and Apache HDFS to ingest and store data. We envision
-instead a unification of the two concepts and our work focuses on both ingesting
-and storing stream data.
-
-Pravega approaches streaming applications from a storage perspective. It enables
-applications to ingest stream data continuously and storing it permanently. Such
-stream data can be accessed with low latency (order of milliseconds), but also
-months, years ahead as part of analyzing historical data.
-
-The design of Pravega incorporates lessons learned from using the Lambda architecture
+Pravega was designed to support the new generation of streaming applications.
+Applications that deal with a large amount of data arriving continuously that
+needs to generate an accurate analysis of that data by considering the factors like:
+
+- Delayed data,
+- Data arriving out of order,
+- Failure conditions.
+
+There are several Open Source tools to enable developers to build such applications,
+including [Apache Flink](https://flink.apache.org/), [Apache Beam](https://beam.apache.org/), [Spark Streaming](https://spark.apache.org/docs/2.2.0/streaming-programming-guide.html), etc.
+These applications uses the following systems to ingest and store data:
+
+- [Apache Kafka](https://kafka.apache.org/),
+- [Apache ActiveMQ](http://activemq.apache.org/),
+- [RabbitMQ](https://www.rabbitmq.com/),
+- [Apache Cassandra](http://cassandra.apache.org/),
+- [Apache HDFS](https://hadoop.apache.org/).
+
+Pravega envision and focuses on both ingesting and storing stream data. Pravega approaches streaming applications from a storage perspective. It enables applications to ingest Stream data continuously and stores it permanently. Such Stream data can be accessed with low latency (order of milliseconds) and also analyzes historical data.
+
+The design of Pravega incorporates using the [**Lambda Architecture**](http://lambda-architecture.net/)
```

Comment:
No. We allow you to move away from a lambda architecture to a kappa architecture.

Reply:
Reverted to old sentence